DQM/GEM/plugins/GEMEfficiencyAnalyzer.cc

```diff
@@ -786,6 +803,23 @@ const CSCSegment* GEMEfficiencyAnalyzer::findCSCSegment(const reco::Muon& muon,
   }
 }
 
+// https://github.com/cms-sw/cmssw/blob/CMSSW_12_4_0_pre3/DataFormats/MuonDetId/interface/MuonSubdetId.h
```

Comment:
What's this comment intended to? I imagine it can be removed

Reply:
I added a link to let the next developer know what the number means. Is it okay to add more details the next time?